package io.soboro.supreme.core.api.domain.user.exception

class InvalidEmailException : RuntimeException("invalid email has been occurred")
